src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
+    private static final int LINE_NAME = 0;
+    private static final int LINE_VALUE = 1;
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeReportToFile(readFile(fromFileName), toFileName);
```

Comment:
order of methods calls should be the next:
```
 String report createReport(readFile(fromFileName);
 writeReportToFile(report, toFileName);
 ```

Reply:
not fixed, this is important so let's not inline the operations and do this with variables, this will be easier to understand
```suggestion
        List<String> data = readFile(fromFileName);
        String report = createReport(data);
        writeToFile(toFileName, report);
```
We are doing this because of the single responsibility that each method even a class should have
\+ better to use abstract names, since our writeToFile method can write not only some specific reports but any string actually

Reply:
I don`t realy understand with what purpose create String if we can get resulr right from method?